package.json

```diff
@@ -4,9 +4,9 @@
   "dependencies": {
     "glamorous": "^4.11.2",
     "govuk-colours": "^1.0.2",
+    "jest-glamor-react": "^3.3.0",
     "prop-types": "^15.6.0",
-    "react": "^15.0.0",
-    "react-dom": "^15.0.0"
+    "react": "^15.6.2"
```

Comment:
we eventually need to move react in to:

1. devDependencies
2. peerDependencies with a large range

..but this doesn't need to be done now